src/test/compiled_ram_index_utils_test.cpp

```diff
@@ -0,0 +1,178 @@
+/*
+ * Souffle - A Datalog Compiler
+ * Copyright (c) 2013, 2015, Oracle and/or its affiliates. All rights reserved
+ * Licensed under the Universal Permissive License v 1.0 as shown at:
+ * - https://opensource.org/licenses/UPL
+ * - <souffle root>/licenses/SOUFFLE-UPL.txt
+ */
```

Comment:
Afraid not - this code is split off from the Compiled_ram_relations_test.cpp, so the copyright remains with Oracle (correct me if I'm wrong however)